DeleteLLVMComments.java

```diff
@@ -0,0 +1,56 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+import java.util.Scanner;
+
+// Copyright (c) 2015 K Team. All Rights Reserved.
+public class DeleteLLVMComments {
+
+    /**
+     * @author LiyiLi
+     * This java code takes in a LLVM program and
+     * eliminates the LLVM comments in the program, then
+     * outputs the modified LLVM program
+     * to a temporary LLVM file.
+     */
+    public static void main(String[] args) {
+        String fileName = args[0];
+        File llvmFile = new File(fileName);
+        if(llvmFile.isFile()){
+            try {
+                Scanner scanner = new Scanner(llvmFile);
+                PrintWriter newLlvmProgam = new PrintWriter("llvmtemp.ll");
```

Comment:
So you cannot call the parser in parallel on several LLVM programs?  In C, for example, we execute all the test suite concurrently.  This way, it now takes 8 minutes instead of 2h.  We will want to do the same for LLVM.  There are standard ways in Java to get a fresh temporary file name.  However, I am not even sure that passing files is the best way to go ... The `kast` tool can be customized to take a string as argument, so one possibility, probably better, would be to call `kast` from this java program (in a platform-independent way, of course) with the resulting, uncommented LLVM program.as a string argument.  Think about it this way: you want your LLVM semantics to work equally well on Linux, Mac, Windows, and you want to be able to just call ktest on a config.xml file that invokes krun on thousands of programs at the same time.